CryptKeeperZK/crypt-keeper-extension
# feat: integrate web3 react package

## Explanation

Currently, there is no agile approach to connect using different wallets.
This PR is a last part of adding support for multiple wallets as well as fixing hook architecture.

Details are below:
- [x] Get rid of custom web3 providers
- [x] Remove unused code
- [x] Integration fixes for create identity
- [x] Reuse useWallet hook for chain related data
- [x] Remove redux web3 store slice
- [x] Use eager connect strategy for wallet

## More Information

Closes #67
Closes #68
Closes #78
Closes #79
Blocked by #125

## Screenshots/Screencaps

N/A

## Manual Testing Steps

1. Install extension
2. Connect with metamask
3. Chain data and balance should load as before

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
